cmd/build/helmify/static/values.yaml

```diff
@@ -34,6 +34,8 @@ postInstall:
       tag: v3.9.0-beta.0
       pullPolicy: IfNotPresent
       pullSecrets: []
+    extraNamespaces:
+    - kube-system
```

Comment:
Should we default to an empty list to preserve existing behavior?

Reply:
Done.